tests/queries/0_stateless/01086_window_view_cleanup.sh

```diff
@@ -12,38 +11,40 @@ opts=(
     "--allow_experimental_analyzer=0"
 )
 
-$CLICKHOUSE_CLIENT "${opts[@]}" --allow_deprecated_database_ordinary=1 <<EOF
-SET allow_experimental_window_view = 1;
-SET window_view_clean_interval = 1;
+DATABASE_ORDINARY="${CLICKHOUSE_DATABASE}_ordinary"
 
-DROP DATABASE IF EXISTS test_01086;
-CREATE DATABASE test_01086 ENGINE=Ordinary;
+$CLICKHOUSE_CLIENT "${opts[@]}" --allow_deprecated_database_ordinary=1 --multiquery "
+    SET allow_experimental_window_view = 1;
+    SET window_view_clean_interval = 1;
 
-CREATE TABLE test_01086.dst(count UInt64, market Int32, w_end DateTime) Engine=MergeTree ORDER BY tuple();
-CREATE TABLE test_01086.mt(a Int32, market Int32, timestamp DateTime) ENGINE=MergeTree ORDER BY tuple();
-CREATE WINDOW VIEW test_01086.wv TO test_01086.dst WATERMARK=ASCENDING AS SELECT count(a) AS count, market, tumbleEnd(wid) AS w_end FROM test_01086.mt GROUP BY tumble(timestamp, INTERVAL '5' SECOND, 'US/Samoa') AS wid, market;
+    DROP DATABASE IF EXISTS ${DATABASE_ORDINARY};
+    CREATE DATABASE ${DATABASE_ORDINARY} ENGINE=Ordinary;
 
-INSERT INTO test_01086.mt VALUES (1, 1, toDateTime('1990/01/01 12:00:00', 'US/Samoa'));
-INSERT INTO test_01086.mt VALUES (1, 2, toDateTime('1990/01/01 12:00:01', 'US/Samoa'));
-INSERT INTO test_01086.mt VALUES (1, 3, toDateTime('1990/01/01 12:00:02', 'US/Samoa'));
-INSERT INTO test_01086.mt VALUES (1, 4, toDateTime('1990/01/01 12:00:05', 'US/Samoa'));
-INSERT INTO test_01086.mt VALUES (1, 5, toDateTime('1990/01/01 12:00:06', 'US/Samoa'));
-EOF
+    CREATE TABLE ${DATABASE_ORDINARY}.dst(count UInt64, market Int32, w_end DateTime) Engine=MergeTree ORDER BY tuple();
```

Comment:
as I remember, no-parallel tag was added here as a blind attempt to fix its flakyness, e.g. it was possible it could get flaky because server was slow with parallel run because the test depends on time.

Reply:
ah, this test used to create a database, then what I wrote is not the case for this test.

Reply:
I'm worried because of this comment https://github.com/ClickHouse/ClickHouse/issues/65583#issuecomment-2248932802

Reply:
Marking `no-parallel` without understanding (or explaining why in the file) is plain wrong. I'll keep enabling tests that don't have any comment or obvious reason, and disable it if it becomes flaky.

Maybe the problem is a bug in window views that should be fixed. I'd rather disable the tests until it's fixed (or removed) than keep sequential tests (that add CI runtime everywhere).

Reply:
I have a feeling we add or remove `no-parallel` for window view tests every few months.
can we make at least a check in `clickhouse-test` that `no-parallel` tag has a comment?

In the same way as marking test `no-parallel` without a comment is unclear, it's not clear what you are doing.
So let's try to prevent someone adding the tag back in the future without explanation.

Reply:
I'm mostly removing hardcoded names (databases, policies) that can cause collisions. It's why many tests are marked as no-parallel. But I can't explain why it's 100% ok since the reason for `no-parallel` wasn't there to begin with, so I'm just assuming that name collision was the only reason.